docs/admin/notifications.rst

```diff
@@ -61,6 +61,51 @@ please refer to the `Open Zaak authorization documentation`_.
 For more information on how to set up subscriptions in Open Notificaties, please refer
 to the `Open Notificaties subscription documentation`_.
 
+Automatic retry for notifications
+=================================
+
+By default, sending notifications has automatic retry behaviour, i.e. if the notification
+publishing task has failed, it will automatically be scheduled/tried again until the maximum
+retry limit has been reached.
+
+Autoretry explanation and configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Retry behaviour is implemented using binary exponential backoff with a delay factor,
+the formula to calculate the time to wait until the next retry is as follows:
+
+.. math::
+
+    t = \text{backoff_factor} * 2^c
```

Comment:
You should mention that t is time in seconds

Reply:
Also the back off number. Specifically that it is seconds

Reply:
I'll add it for `t`, though I don't know if the backoff factor has a unit, since it's just a delay factor